tests/integration/targets/elasticache/tasks/main.yml

```diff
@@ -0,0 +1,132 @@
+---
+
+- name: Integration testing for the elasticache module
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
```

Comment:
Looks like this whole section isn't working, as params are not applied to my test tasks. What is it I am doing wrong here? I simply took this from other tests.